.travis.yml

```diff
@@ -1,17 +1,37 @@
 language: python
+python:
+    - 2.7
+    - 3.3
+    - 3.4
+    - 3.5
+    - 3.6
+
+cache: pip
+
+env:
+    - TOXENV=django18
+    - TOXENV=django110
+    - TOXENV=django111
+
+matrix:
+    exclude:
+    - python: 3.6
+      env: TOXENV=django18
+    - python: 3.3
+      env: TOXENV=django110
+    - python: 3.6
+      env: TOXENV=django110
+    - python: 3.3
+      env: TOXENV=django111
+    include:
+    - python: 3.6
+      env: TOXENV=quality
 
 install:
-    - pip install -r requirements/base.txt
```

Comment:
The only requirement in base.txt is `django`, which we want to install via `tox` on Travis. No point in installing it here, then overwriting later!